source/agora/consensus/data/Block.d

```diff
@@ -449,6 +450,7 @@ public Block makeNewBlock (Transactions)(const ref Block prev_block,
     block.txs.sort;
 
     block.header.merkle_root = block.buildMerkleTree();
+    enrollments.each!(enrollment => block.header.enrollments ~= enrollment);
```

Comment:
`block.header.enrollments = enrollments` should be good enough.

Reply:
applied the suggestion.